### ISSUE TITLE
Tayari implementation issues - Lesson Ended and Consent

### ISSUE THREAD

**Issue by chrisekelley** [Tangerine-Community/Tangerine-client#261]
From: Brian Njamwea 
Date: Friday, February 5, 2016 at 9:21 AM

Subject: Re: FW: TEST RESULTS 03/02/15: Tayari Project - Tangerine on Android 5.0

Dear Scott & Kelley,

Okay, thank you.

After the Quality Control team tested the latest version of Tayari on Tangerine for Android 5.0, the APK & blank screen errors introduced after yesterday's change no longer occur. However, the following originally reported issues for fixing as per the attachment are still occurring:
1. The ‘Lesson Ended” button in the Numeracy and literacy Observation does not work. 
2. The "Consent" for school questionnaire doesn't allow for reason for not participating with the interview.

Please progress.

Thank you.

---

addl info:

From: Brian Njamwea 
Date: Friday, February 5, 2016 at 9:54 AM
Subject: Re: TEST RESULTS 03/02/15: Tayari Project - Tangerine on Android 5.0

Okay.

For Issue 2, Scott had correctly indicated we could capture reason for refusal via 'Additional Comments'. However, if an additional field could be added to the "Consent" sub-test to specifically capture this and thereby separate the this reason from general comments, it would be great. In the meantime, I can treat this as a training issue as previously suggested pending your revert.

I look forward to your feedback on the 2 issues logged.

Do have a great weekend.

**Comment by skippskipp** [comment]
I disagree with the request for adding an additional field for comment when "consent" == no.  Such a field would bear no substantive difference from the current "Additional Comments" field.
